.github/workflows/test.yml

```diff
@@ -0,0 +1,11 @@
+name: "Test"
+on:
+  pull_request:
+  push:
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v7
+    - run: nix-build -I nixpkgs=https://github.com/NixOS/nixpkgs-channels/archive/nixpkgs-unstable.tar.gz 
```

Comment:
Might be redundant but at least contributors can just copy and paste the same line to get the same result (tm).